homework-submission/todo-app/src/Components/Form.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import '../App.css';
+import DatePicker from 'react-date-picker';
+
+class Form extends Component {
+    state = {
+        description: '',
+        deadline: new Date()
+    };
+
+    handleChange = evt => {
+        this.setState({ description: evt.target.value })
+    };
+
+    handleDateChange = date => {
+        this.setState({ deadline: date });
+
+    };
+
+    handleSubmit = evt => {
+        evt.preventDefault();
+        this.props.onAdd(this.state.description, this.state.deadline);
+        this.setState({ description: '', deadline: new Date() });
+    };
+
+    render() {
+        return (
+            <div className="Form">
+                <form onSubmit={this.handleSubmit}>
+                    <input
+                        className="description"
+                        type='text'
+                        placeholder="Insert a todo"
+                        onChange={this.handleChange}
+                        value={this.state.description}
+
+                    />
+                    <DatePicker
+                        className="date_picker"
+                        value={this.state.deadline}
+                        onChange={this.handleDateChange}
+                        minDate={new Date()}
+                    />
+                    <button type="submit">
```

Comment:
While you are using the state to follow the input values, using for is not needed, you can add: 
```suggestion
                    <button onClick={this.handleSubmit}>
```

Then no need for the `type="submit"` any more because it's not a form 😃